dev1 updates to support RRFS 13-km NA

jaymes-kenyon opened the pull request:
## DESCRIPTION OF CHANGES: 
This PR provides several changes in support of the RRFS 13-km NA domain.  This domain is currently regarded as a separate configuration of the dev1 codes.  These updates provide:
(1) a unique config.sh file 
(2) modifications to scripts that retrieve real-time GFS GRIB2 files from /public on Jet (for IC/BC processing)
(3) a configurable variable ( EXTRN_MDL_LBCS_FILE_NUM ) to help generalize the dependency for the task "get_extrn_lbcs"   
(4) updates to grid specifications for the write component (from Gerard K.)

## TESTS CONDUCTED: 
A prototype configuration using these codes is currently running in real time.  All tasks through "run_fcst" appear to be completing successfully.  However, subsequent changes may be required to improve stability of "run_post" tasks.

jaymes-kenyon committed:
Updates to dev1 regional_workflow to enable/support 13-km North America variant.

jaymes-kenyon committed:
Adding a configurable variable "EXTRN_MDL_LBCS_FILE_NUM", used in the dependency for get_extrn_lbcs.

christinaholtNOAA commented on ush/config.sh.RRFS_NA_13km:
```diff
@@ -0,0 +1,87 @@
+ACCOUNT="nrtrr"
+RESERVATION="rrfsdet"
+EXPT_BASEDIR="/home/rtrr/RRFS"
+EXPT_SUBDIR="RRFS_NA_13km"
```
This should be `RRFS_NA_13km_dev1` for consistency with RRFS_AK.

christinaholtNOAA commented on ush/config.sh.RRFS_NA_13km:
```diff
@@ -0,0 +1,87 @@
+envir="para"
+
+NET="RRFS_NA_13km"
+TAG="RRFS_NA_13km"
```
TAG should include "dev1", so maybe "RRFS_dev1_NA_13km" to follow what's happening with CONUS and AK now.

jaymes-kenyon replied:
Will do—what about "NET"?  I'm not sure what it does.

christinaholtNOAA replied:
It looks like NET sets the prefix for the grib files out of post, and is included as part of the PTMP directory structure under `com`.

christinaholtNOAA commented on ush/config.sh.RRFS_NA_13km:
```diff
@@ -0,0 +1,87 @@
+
+USE_CUSTOM_POST_CONFIG_FILE="TRUE"
+CUSTOM_POST_CONFIG_FP="/mnt/lfs4/BMC/nrtrr/RRFS/dev1_13km_NA-ufs-srweather-app/src/EMC_post/parm/postxconfig-NT-fv3lam_rrfs.txt"
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_na_13km"
```
Should include `_dev1` as a suffix.

christinaholtNOAA commented on ush/config.sh.RRFS_NA_13km:
```diff
@@ -0,0 +1,87 @@
+ARCHIVEDIR="/5year/BMC/wrfruc/rrfs_na_13km"
+NCARG_ROOT="/apps/ncl/6.5.0-CentOS6.10_64bit_nodap_gnu447"
+NCL_HOME="/home/rtrr/RRFS/graphics"
+NCL_REGION="conus"
```
Should this be "north america"? I'm not sure since it's "conus" and "alaska" for the other two runs.

jaymes-kenyon replied:
Looks like our two NCL domain options are currently "conus" and "alaska".  May leave this as "conus" for now, but return to the issue of graphics production shortly.

christinaholtNOAA commented on ush/config.sh.RRFS_NA_13km:
```diff
@@ -0,0 +1,87 @@
+#RUN_TASK_MAKE_OROG="FALSE"
+#RUN_TASK_MAKE_SFC_CLIMO="FALSE"
+
+RUN="RRFS_NA_13km"
```
Suffix with "dev"?

christinaholtNOAA reviewed (approved):
The code looks good. I have a few questions/comments about the configurable values below, mainly related to adding "dev1" tags for clarity of output. When those are changed, I think it's ready.

jaymes-kenyon committed:
Update config.sh.RRFS_NA_13km

Updates to various naming conventions; thanks to @christinaholtNOAA for helpful guidance.
